### PR TITLE
Remove DCDO from congrats

### PR DESCRIPTION
**Pull request by Erin007**
Initially, we used a DCDO flag to ensure that Code.org tutorials redirected to studio.code.org/congrats correctly rather than to the original code.org/congrats. The flag is no longer necessary so this PR removes it. 
  
Additionally, this change includes updates to the hocOfCodeFinish UI test.  These changes were necessary because the test environment doesn't use DCDO and because the image checks done previously now have additional numbers appended to their url since they are coming in through the JS pipeline via React.

### CODE REVIEW

File: pegasus/helpers/hoc_helpers.rb [hunk 1]
```diff
@@ -123,7 +123,7 @@ def complete_tutorial(tutorial={})
     end
 
     site = "http://#{row[:referer]}"
-    if DCDO.get('new_congrats', false) && tutorial[:orgname].try(:include?, 'Code.org')
+    if tutorial[:orgname].try(:include?, 'Code.org')
       site = CDO.studio_url('', CDO.default_scheme)
     end
```
Comment (breville) [inline pegasus/helpers/hoc_helpers.rb]:
```
else
  site = "http://#{row[:referer]}"
```
?

Comment (breville) [reply pegasus/helpers/hoc_helpers.rb]: i.e. we only need to set site once, rather than conditionally overwrite it.

**Comment by joshlory** [comment]
UI test still failing in CircleCI?

**Comment by Erin007** [comment]
Yes, it's still failing on Circle 😞.  It passes locally and everything looks good when I run through the steps manually.  Haven't figured out what exactly is different about the test on Circle yet. Any ideas?

**Comment by joshlory** [comment]
The visual log shows a "Bad Gateway" error that I don't recognize: https://saucelabs.com/beta/tests/489d24b5624d4d5e880beba7d8aa8728/commands#26.  (I found this by viewing the [Artifacts for the failed CircleCi run](https://circleci.com/gh/code-dot-org/code-dot-org/57208#artifacts/containers/0), and locating [hourOfCodeFinish_output.html](https://57208-24250009-gh.circle-artifacts.com/0/ui-test-logs/ChromeLatestWin7_hourOfCodeFinish_output.html).)

I wonder if we're getting an https URL instead of http in UI tests, which is failing to resolve?

**Comment by Erin007** [comment]
`default_scheme` is important, because without it the url appends like this: 
screen shot 2018-01-09 at 10 47 09 am

**Comment by joshlory** [comment]
One question to ask: `replace_hostname` in UI tests should already handle the https -> http conversion.  Why is that not happening here?

https://github.com/code-dot-org/code-dot-org/blob/d70a67a3dadd4ddb4345e43460f9ef68717a5061/dashboard/test/ui/features/step_definitions/steps.rb#L56-L59

File: apps/src/templates/Certificate.jsx [hunk 2]
```diff
@@ -145,7 +146,7 @@ class Certificate extends Component {
           )}
           {tutorial && this.state.personalized && (
             <div>
-              <h2>{i18n.congratsCertificateThanks()}</h2>
+              <h2 id="thanks">{i18n.congratsCertificateThanks()}</h2>
```
Comment (joshlory) [inline apps/src/templates/Certificate.jsx]: Elsewhere we've used [`uitest-`](https://github.com/code-dot-org/code-dot-org/search?q=uitest) as a prefix for IDs to make it clear that they're only used for testing, not for CSS or JS lookup.

**Comment by tanyaparker** [comment]
[Make new_congrats DCDO flag permanent.](https://trello.com/c/lxauW6zb/41-make-newcongrats-dcdo-flag-permanent)

**Pull request merged**